backend/Recodo.BLL/Services/FileService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Azure.Storage.Blobs;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Recodo.DAL.Context;
+using Recodo.DAL.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recodo.BLL.Services
+{
+    public class FileService
+    {
+        private readonly RecodoDbContext _context;
+
+        public FileService(RecodoDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<int> SaveVideo(int authorId)
+        {
+            Video newVideo = new Video();
```

Comment:
This way please 
            var newVideo = new Video() {
            IsSaving = true,
            AuthorId = authorId,
            ...
            }